### PR TITLE
CA-90 & CA-91 Cloudwatch log collection for HBase and Spark | CA-134 Deprecates s3 policy arns in EMR module

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
* Adds example usage of cloudwatch log collection in examples/static-hbase-logs.
* Adds example usage of cloudwatch log collection in examples/static-spark-logs.
These examples will deploy the necessary resources on every instance that belongs to an EMR Cluster in order to:

Download the Cloudwatch agent.
Install the Cloudwatch agent.
Create and mount the configuration file.
Enable the Cloudwatch agent.

In this module we also:

Deprecates s3_policy_arns in favor of additional_policy_arns.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # TAMR AWS EMR module
+# v7.3.0 - January 11st, 2022
```
Comment (franco-martin) [inline CHANGELOG.md:2]: January 11th

File: examples/static-hbase-logs/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+This example depends on an existing VPC and subnet with configurations that meet [EMR cluster requirements](https://aws.amazon.com/blogs/big-data/launching-and-running-an-amazon-emr-cluster-inside-a-vpc/).
```
Comment (franco-caylent) [inline examples/static-hbase-logs/README.md:1]: that meetS

File: examples/static-hbase-logs/emr.tf [hunk 3]
```diff
@@ -0,0 +1,78 @@
+locals {
```
Comment (franco-caylent) [inline examples/static-hbase-logs/emr.tf:1]: you'll need to run tf fmt --recursive

**Review by franco-caylent — COMMENTED** [review]
LGTM

File: examples/static-spark-logs/variables.tf [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+variable "compute_subnet_id" {
+  type        = string
+  description = "ID of the subnet where the EMR cluster will be created. If `abac_valid_tags` key values are set, this subnet is required to have a valid key value tag as well."
+}
+
+variable "vpce_logs_endpoint_dnsname" {
+  type        = string
+  description = "DNS name of the Cloudwatch VPC Interface Endpoint which will be provided to the script to install and configure the Cloudwatch agent."
+}
```
Comment (souza-dan) [inline examples/static-spark-logs/variables.tf:60]: Is this currently an output of the networking module?
https://github.com/Datatamer/terraform-aws-networking/pull/13

If not could it be?

File: examples/static-hbase-logs/networking.tf [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+module "aws-emr-sg-master" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = []
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports.ingress_master_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "-master")
+  egress_protocol         = "-1"
+  ingress_protocol        = "6"
```
Comment (souza-dan) [inline examples/static-hbase-logs/networking.tf:16]: Similar comment to the other PR, but what motivated using 6 instead of "tcp"?

**Review by souza-dan — APPROVED** [review]
LGTM, great examples and documentation

**Comment by fvazquez-caylent** [comment]
https://github.com/Datatamer/terraform-aws-emr/pull/68#discussion_r783291888

For sure. I already added it and will be visible in the next commit.
Thanks!

**Review by souza-dan — APPROVED** [review]

**Pull request merged**